lib/github/github.ex

```diff
@@ -188,6 +201,25 @@ defmodule BorsNG.GitHub do
     user
   end
 
+  @spec get_team_by_name(tconn, bitstring, bitstring) ::
```

Comment:
Please, instead of bitstring use `String.t`

Reply:
Note: this everywhere that `bitstring` appears

Reply:
⚡️  I've changed everywhere I've added a `bitstring`.  I'm leaving the other bitstrings alone.